SPECS/kernel/config_aarch64

```diff
@@ -527,7 +527,6 @@ CONFIG_ARM64_EPAN=y
 # end of ARMv8.7 architectural features
 
 CONFIG_ARM64_SVE=y
-CONFIG_ARM64_SME=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/22aaaa7a1a732a5288aa4a5785a77b6895959134

ARM Scalable Matrix Extension (SME) is being dubbed "BROKEN" for now due to many issues including the corruption of the FPSIMD/SVE/SME state of arbitrary tasks.

More on ARM SME can be read at
[KConfig](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-3/6.6.62.1/arch/arm64/Kconfig#L2166)
[linux kernel upstream documentation](https://www.kernel.org/doc/html/next/arm64/sme.html)
[ARM blog](https://community.arm.com/arm-community-blogs/b/architectures-and-processors-blog/posts/scalable-matrix-extension-armv9-a-architecture)